packages/ui-components/src/components/Inputs/ImageUploadField.js

```diff
@@ -79,7 +79,7 @@ const LabelWrapper = styled(Box)`
 export const ImageUploadField = React.memo(
   ({
     name,
-    encodedImage,
+    imageSrc,
```

Comment:
this is a good rename. I was confused when I was reading encodedImage earlier today since it's a url sometimes